src/app/lib/config/services/persian.js

```diff
@@ -7,6 +7,7 @@ const persian = {
   defaultImageAltText: 'BBC News فارسی',
   externalLinkText: ' ،لینک خارجی',
   imageCaptionOffscreenText: ' ، عنوان تصویر',
+  videoCaptionOffscreenText: ' ، عنوان تصویر',
```

Comment:
```suggestion
  videoCaptionOffscreenText: ' ، عنوان ویدئو',
```
Translation suggestion.